v7/import_blogger/import_blogger.py

```diff
@@ -191,12 +194,16 @@ def import_item(self, item, out_folder=None):
             # If no content is found, no files are written.
             content = self.transform_content(content)
 
-            self.write_metadata(os.path.join(self.output_folder, out_folder,
-                                             slug + '.meta'),
-                                title, slug, post_date, description, tags)
-            self.write_content(
-                os.path.join(self.output_folder, out_folder, slug + '.html'),
-                content)
+            link_fragments = link_path.split('/')
+            if len(link_fragments) == 4:
```

Comment:
* this check is unnecessary; however, you should have `slug` pre-determined and set to the correct value much earlier (line 161)
* remove the unnecessary `re` import